### ISSUE TITLE
The PVAM module is different with paddleOcr

### ISSUE THREAD

**Issue by archwolf118** [chenjun2hao/SRN.pytorch#11]

**Comment by archwolf118** [comment]
I try to write the code by myself:

class PVAM(nn.Module):
    ''' Parallel Visual attention module 平行解码'''
    '''
    n_dim：512，阅读顺序序列编码的空间维度
    N_max_character: 25，单张图片最多有多少个字符
    n_position: cnn出来之后特征的序列长度
    '''
    def __init__(self,  n_dim=512, N_max_character=25, n_position=256):

        super(PVAM, self).__init__()
        self.character_len = N_max_character

        self.wv = nn.Linear(n_dim, n_dim)
        self.we = nn.Linear(n_dim, 1)
        self.softmax = nn.Softmax(dim=2)
        self.f0_embedding = nn.Embedding(N_max_character, n_dim)
        self.tanh = nn.Tanh()

    def forward(self, word_features):
        enc_output = word_features
        b, t, c = word_features.shape

        word_features = word_features.contiguous().view(b * t, c)    
        word_features = self.wv(word_features)
        word_features = word_features.view(-1, 1, t, c)
        word_features = word_features.expand(-1, self.character_len,-1, -1)

        reading_order = torch.arange(self.character_len, dtype=torch.long, device=enc_output.device)
        reading_order = reading_order.unsqueeze(0).expand(enc_output.size(0), -1)
        reading_order_embed = self.f0_embedding(reading_order)
        reading_order_embed = torch.unsqueeze(reading_order_embed, dim=2)
        reading_order_embed = reading_order_embed.expand(-1, -1, t, -1)

        temp = word_features + reading_order_embed
        temp = self.tanh(temp)
        temp = temp.contiguous().view(b*self.character_len*t, c)

        attention_weight = self.we(temp)
        attention_weight = attention_weight.view(b, self.character_len, t)
        attention_weight = self.softmax(attention_weight)

        g_output = torch.bmm(attention_weight, enc_output)  # b,25,512
        return g_output

But it cannot converge, I don't know where is the problem?
Please pay attention to this problem, because the key point of SRN is the PVMA module, In the paper, without the GSRM, the model also perform good result.
